src/ng-generate/components/shared/generators/components/command-bar/index.ts

```diff
@@ -100,3 +92,16 @@ function getPropertiesToCreateFilters(options: any, allProps: Array<Property>):
 function datePickerType(datePickers: Array<DatePicker>, propertyValue: PropValue): string | undefined {
     return datePickers.find((value: any) => value.propertyUrn === propertyValue.propertyUrn)?.datePicker.type;
 }
+
+function sortItemsByArray(orderArray: string[], itemsArray: Property[]): Property[]{
```

Comment:
You can also say: 

`function sortItemsByArray(orderArray: string[], itemsArray: Property[]): Property[] {
    const orderMap = orderArray.reduce((map, name, index) => ({ ...map, [name]: index }), {});
    return itemsArray.sort((a, b) => orderMap[a.name] - orderMap[b.name]);
}`

Reply:
done